apps/web/src/server/applications/create-new-case/key-dates/applications-create-key-dates.validators.js

```diff
@@ -1,60 +1,41 @@
 import { createValidator } from '@pins/express';
```

Comment:
I reused the date validator from the "key-dates" step of the "create-new-case" journey.

To do so I had to standardise the error messages. They make sense to me, but I guess testers will approve them.